src/mimeparser.rs

```diff
@@ -136,6 +137,14 @@ const MIME_AC_SETUP_FILE: &str = "application/autocrypt-setup";
 
 impl MimeMessage {
     pub async fn from_bytes(context: &Context, body: &[u8]) -> Result<Self> {
+        MimeMessage::from_bytes_with_partial(context, body, None).await
+    }
+
+    pub async fn from_bytes_with_partial(
```

Comment:
Documentation comment is really needed, completely impossible to guess what `partial` means otherwise.